tests/extension/oneapi-current-device/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(SYCL_CTS_ENABLE_EXT_ONEAPI_ROOT_GROUP_TESTS)
```

Comment:
Naming scheme of all other folders in `tests/extension/` folder is different: there are `_` instead of `-`, please, rename the folder